nix/runtime.nix

```diff
@@ -24,9 +26,9 @@ rec {
       # Postgres will always be accessible via `postgres:5432` from
```

Comment:
Maybe update this comment to be clearer

Reply:
I think we should move all of it to the template. It's mostly outdated.

Reply:
Not in this issue I mean.